src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonSchemaCreateOptions.cs

```diff
@@ -37,15 +37,24 @@ public sealed record class AIJsonSchemaCreateOptions
     /// </remarks>
     public Func<ParameterInfo, bool>? IncludeParameter { get; init; }
 
+    /// <summary>
+    /// Gets a <see cref="AIJsonSchemaTransformOptions"/> governing transformations on the JSON schema after it has been generated.
+    /// </summary>
+    public AIJsonSchemaTransformOptions? TransformOptions { get; init; }
+
     /// <summary>
     /// Gets a value indicating whether to include the type keyword in inferred schemas for .NET enums.
     /// </summary>
+    [Obsolete("This property has been deprecated.")]
+    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
     public bool IncludeTypeInEnumSchemas { get; init; } = true;
```

Comment:
I don't see an equivalent for IncludeTypeInEnumSchemas in transform options. I see it's defaulting to true, so presumably we're emitting the information, and then if someone didn't want it they'd write a transform to remove it? It's not common enough to need being exposed on the transform options?

Reply:
https://github.com/dotnet/extensions/pull/6427#discussion_r2085159460

Reply:
Adding the `type` keyword to enums is redundant, but it's also benign. There is no reason why we'd want to switch this off.